integreat_cms/cms/templates/pages/page_form.html

```diff
@@ -167,6 +167,10 @@ <h1 class="heading">
                 </div>
             </div>
             <div class="flex flex-wrap flex-col gap-4">
+                {% if language.slug == "de" and request.region.hix_enabled %}
```

Comment:
Maybe it makes sense to add a setting for which language slugs hix should be enabled (I can think of e.g. enabling it for simple german as well in the future...)
```suggestion
                {% if language.slug in TEXTLAB_API_LANGUAGES and request.region.hix_enabled %}
```